scripts/start.sh

```diff
@@ -9,10 +9,10 @@ echo 'Enter y or n'
 read response
 
 if [ $response == "n" ]; then
-  source venv/bin/activate
+  source ../venv/bin/activate
```

Comment:
Nice work @parseendavid, But I don't think changing the paths to file in the bash scripts is necessary and I don't even think it will make the scripts run successfully from the root, did you try running the commands again from the root to be sure it's working? The same applies to other instances...

Reply:
Was this actually tested @ayodelevm

Reply:
Was this actually tested @parseendavid

Reply:
my bad when we were testing I was running the commands from within the script directory....